app/src/main/java/com/concordium/wallet/data/util/CurrencyUtil.kt

```diff
@@ -9,6 +9,8 @@ import java.text.DecimalFormatSymbols
 import java.util.regex.Pattern
 
 object CurrencyUtil {
+    val MAX_AMOUNT = "115792089237316195423570985008687907853269984665640564039457584007913129639935".toBigInteger()
```

Comment:
Can you please document where this is coming from?

Thanks.

Reply:
As far as I know, it is the max possible amount in Concordium, the max value of uint256. If it is correct, I'll add the corresponding comment to this constant.

Reply:
Yes, that is correct. I just wanted you to document this, that's all :)